.github/workflows/reusable-e2e.yml

```diff
@@ -348,7 +348,7 @@ jobs:
   e2e-ios:
     needs: check-changed-packages
     if: ${{ inputs.skip-changed-packages-check == 'true' || needs.check-changed-packages.outputs.has-changed-packages == 'true' }}
-    runs-on: macos-13
+    runs-on: macos-12
```

Comment:
I know we probably need to do this given the issues with the runner, but MacOS 12 was slower, right?